biapol_utilities/label/_filter_similarity_matrix.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+
+def suppressed_similarity(matrix, threshold=0.25):
```

Comment:
I think there is no test for this function. Would you mind adding one?

Reply:
Added in 290dc6408d5677cd25222cfd0d6d6f793aff5572